frontend/javascripts/types/api_flow_types.ts

```diff
@@ -779,6 +778,16 @@ export type ServerSkeletonTracingTree = {
   isVisible?: boolean;
   type?: TreeType;
   edgesAreVisible?: boolean;
+  // todop: check whether this is really not-optional
+  metadata: MetadataEntry[];
+};
+export type MetadataEntry = {
+  key: string;
+  stringValue?: string;
+  boolValue?: boolean;
+  numberValue?: number;
+  // todop: the server always sends an empty array
```

Comment:
maybe you can answer this, @fm3?

Reply:
works for me :thinking: let’s have a look together